gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-agp = "7.4.1" # 7.4.1 is the maximum version supported by IDEA 2023.1
+agp = "8.0.1" # 7.4.1 is the maximum version supported by IDEA 2023.1
```

Comment:
@cedrickcooke where did you find the AGP / IDEA version compatibility?
I see that [IDEA 2023.1.1 is out](https://www.jetbrains.com/idea/download/#section=mac), but I'm having trouble figuring out if it supports this bump (AGP 8.0.1).